### ISSUE TITLE
Change the Daily Puzzle in the Homepage to Daily Tsume.

### ISSUE THREAD

**Issue by Eventlesstew** [WandererXII/lishogi#813]
![image](https://github.com/WandererXII/lishogi/assets/161836208/141b0f89-3609-467c-9581-c1992009f6dd)
I feel like this should be changed to be a Daily Tsumeshogi instead considering that Tsumeshogi is more native to Shogi rather than Puzzles.

**Comment by WandererXII** [comment]
That is true, but moving forward we will be getting constant supply of game puzzles, but no guaranteed supply of new tsume puzzles, so we would probably run out.
